src/send/send_caliper_events.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import fileinput
+import json
+import time
+
+
+def api_handler(data):
+    AUTHORIZATION = 'Authorization'
+    BASIC = 'Basic '
+    MIME_TYPE_JSON = 'application/json'
+    CONTENT_TYPE = 'Content-type'
+    headers = {CONTENT_TYPE: MIME_TYPE_JSON, AUTHORIZATION: BASIC + TOKEN}
+    response = None
+    try:
+        response = requests.post(url, data=data, headers=headers)
+    except (requests.exceptions.RequestException, Exception) as exception:
+        failure_resp.write("Connection to LRS failed %s\n" % exception)
+
+    return response
+
+
+print ("Start Of Script")
+
+TOKEN = 'xxxx'
+url = 'https://xxxx.org'
+fileToRead = '<absolute-path-to-file>'
+
+success_resp = open('success_response.txt', 'w')
+failure_resp = open('failed_response.txt', 'w')
+failed_events = open('failed_events.json', 'w')
+
+success_count = 0
+failure_count = 0
+events_count = 0
+
+caliper_events = open(fileToRead)
+fileContent = caliper_events.read()
+# TODO take delimiter between event data from command line, for this instance of the script assumes it to be 'BREAK'
+events = fileContent.split("BREAK\n")
+start = time.time()
```

Comment:
You mentioned it takes two hours to process about 60,000 events.  I wonder how much of that time is used for reading in the whole file and splitting it on the delimiter.  Other methods might not make much of a difference, but I wonder whether reading in the whole file is the most efficient method.